modules/oe_content_event/config/install/core.entity_form_display.event_profile.oe_event_partner.default.yml

```diff
@@ -57,4 +44,6 @@ content:
     third_party_settings: {  }
     type: link_default
     region: content
-hidden: {  }
+hidden:
```

Comment:
The same changes should probably apply to other entities that are used in inline entity forms? Not a requirement but maybe worth a shot.

Reply:
It might be, for now I just applied here within the scope of the ticket. As soon as we receive the requirements for the back-end form, we will change the Venue as well.